src/main/resources/db/migration/V19__add_created_at_column_to_db_info_table.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE db_table_infos RENAME TO db_table_info;
```

Comment:
this seems unrelated -- does the `DbTableInfoDao` have to change now? Also seems like we don't have `DbTableInfoDao` tests to catch this. Filed #485 

The DAO method below might need to be updated?
```
  @SqlUpdate(
      "INSERT INTO db_table_infos (uuid, db_name, db_schema_name) VALUES (:uuid, :db, :dbSchema)")
  void insert(@BindBean DbTableInfoRow dbTableInfoRow);
```

Reply:
Sure, let's update the table name when we fix #485. Thanks for filling the issue, @sshah-wework!